### DISCUSSION TITLE
Support for Unreal Engine - 5

### DISCUSSION THREAD

**Discussion by canhaifuyun** [gildor2/UEViewer#263] [General]
A good tool, but it doesn't support Unreal Engine - 5.

**Comment by gildor2** [comment]
The engine actually hasn't been released yet. May be later I'll add support (without Nanite perhaps), if it won't require too much time.

**Reply by JMVRy** [reply]
any plans on work now that it has officially released?

**Reply by gildor2** [reply]
I think there's a custom build on the forum (spiritovod's one) which has UE5 support.

**Reply by Wobbley** [reply]
Do you have some links to this one, I was unable to find it through google? Want to test it on some UE5 games.

**Reply by gildor2** [reply]
Perhaps you'll need to find the game in the forum, which is built using UE5. I don't remember anything myself, I'm a rare guest on my own forum now, just keeping it clean and organized.

**Reply by Beivo** [reply]
most games are switching unreal engine 5 now, i think its time to make the new version

**Reply by gildor2** [reply]
Here you may find a custom build with UE5 support:
https://www.gildor.org/smf/index.php/board,44.0.html

**Reply by Beivo** [reply]
only works for 5.0 from what i have found, nothing higher

**Reply by gildor2** [reply]
Here you may find more things about technical difficulties with UE5:
https://www.gildor.org/smf/index.php/topic,7932.0.html

**Reply by Official-Husko** [reply]
Hoping for UE5 support too and thanks for the tool

**Reply by Gerigory** [reply]
looking forward to a version that supports UE5.3 or higher. Just out of curiosity, why not make this tool a external program to unreal engine just like unrealPakViewer, so as to upgrade with Unreal with little cost.

**Reply by gildor2** [reply]
> why not make this tool a external program to unreal engine just like unrealPakViewer, so as to upgrade with Unreal with little cost.

When I've started working on this tool, there was no public source code of ANY unreal engine. Even more, UE4 didn't exist. And UE3 didn't exist. There was just Unreal Tournament 2003, as a ready to use game.

If someone will consider building a model viewer based on the UE source code, as an external program - he'll find out that things are working for SINGLE engine version, exactly which source he'll use, and won't work even with another version of vanilla engine. And no way to open games which has customizations to model data.

**Reply by GatoNaAbobora** [reply]
is there going to be a UE5 or higher or is the project maintenance stopped?

**Reply by beedell-roke** [reply]
> https://github.com/gildor2/UEViewer/discussions/263#discussioncomment-4473513

@gildor2, https://www.gildor.org/smf/index.php/topic,7932.0.html is 404.
